OpenMage/magento-lts
# Allowed to upload images in WebP format

### Description (*)
Since the webp format is now supported by all modern browsers *, I have added the ability to upload images in webp format.
* https://caniuse.com/webp

This PR allow to upload an image in webp format and add support to resize and cache it correctly in frontend.

PS: I know that there are extensions/techniques to dynamically generate the WebP version on the frontend, but based on the target audience, with this PR, we can decide to upload them directly in the correct format.

### Manual testing scenarios (*)
Test all scenarios with an image in webp format

Tested scenarios:
- [x] Product images
- [x] Product Custom Options Input Type: File
![immagine](https://github.com/OpenMage/magento-lts/assets/5071467/5c394701-4acf-4c26-a34a-15ccca1f1bf3)
- [x] Category Image and Thumbnail Image
- [x] CMS WYSIWYG Media Storage

Untested scenarios:
- [ ] Import/export

I create this pull request as a draft since there are areas that I have not yet tested.
Tested on PHP 7.4.29 with GD 2.1.0 (support to webp should be enabled by default)
